### PR TITLE
Denali Input Changes

### PR DESCRIPTION
**Pull request by ayeswarya**
Screen Shot 2021-01-28 at 1 28 23 PM
base_column_config
metric_label
goal_gauge
number_format_selector

Resolves #

## Proposed Changes
Denali changes to Inputs 

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/core/tests/acceptance/table-test.ts [hunk 1]
```diff
@@ -78,8 +78,8 @@ module('Acceptance | table', function(hooks) {
     await visit('/table');
 
     await click('.table-config__total-toggle-button .x-toggle-btn');
-    await fillIn('.timeDimension > .table-header-cell__input', 'test');
-    await blur('.timeDimension > .table-header-cell__input');
+    await fillIn('.timeDimension > .input > .table-header-cell__input', 'test');
```
Comment (jkusa) [inline packages/core/tests/acceptance/table-test.ts:81]: do we need this extra class?

Does ths work?
```suggestion
    await fillIn('.timeDimension .table-header-cell__input', 'test');
```

File: packages/core/tests/acceptance/table-test.ts [hunk 2]
```diff
@@ -78,8 +78,8 @@ module('Acceptance | table', function(hooks) {
...
+    await blur('.timeDimension > .input > .table-header-cell__input');
```
Comment (jkusa) [inline packages/core/tests/acceptance/table-test.ts:82]: Do we need this now that we are triggering on `"input"`

File: packages/core/addon/components/navi-visualization-config/metric-label.hbs [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
-      @type="text"
-      @value={{readonly this.metricColumn.displayName}}
-      @enter={{fn this.updateLabel}}
-      @focus-out={{fn this.updateLabel}}
-      @placeholder="No Label"
+  <DenaliInputGroup @label="Label" @isStacked= {{true}} class="metric-label-config__section-header">
+    <DenaliInput
+      @size="medium"
+      type="text"
+      class="label"
```
Comment (jkusa) [inline packages/core/addon/components/navi-visualization-config/metric-label.hbs:7]:
```suggestion
      class="metric-label-config__label-input"
```

File: packages/core/addon/components/navi-visualization-config/goal-gauge.hbs [hunk 4]
```diff
@@ -1,50 +1,36 @@
...
-      @value={{readonly this.metricColumn.alias}}
-      @focus-out={{fn this.updateLabel}}
-      @key-up={{fn this.updateLabel}}
-      @placeholder={{this.metricColumn.displayName}}
+
+   <DenaliInputGroup @label="Label" @isStacked= {{true}} class="goal-gauge-config__section-header">
+    <DenaliInput
+      @size="medium"
+      type="text"
+      class="label"
```
Comment (jkusa) [inline packages/core/addon/components/navi-visualization-config/goal-gauge.hbs:8]:
```suggestion
      class="goal-gauge-config__label-input"
```

File: packages/core/addon/components/navi-visualization-config/goal-gauge.hbs [hunk 5]
```diff
@@ -1,50 +1,36 @@
...
-      @value={{readonly @options.baselineValue}}
-      @focus-out={{fn this.updateBaseline}}
-      @key-up={{fn this.updateBaseline}}
+  </DenaliInputGroup>
+
+  <DenaliInputGroup @label="Baseline" @isStacked= {{true}} class="goal-gauge-config__section-header">
+    <DenaliInput
+      @size="medium"
+      type="text"
+      class="baseline"
```
Comment (jkusa) [inline packages/core/addon/components/navi-visualization-config/goal-gauge.hbs:19]:
```suggestion
      class="goal-gauge-config__baseline-input"
```

File: packages/core/addon/components/navi-visualization-config/goal-gauge.hbs [hunk 6]
```diff
@@ -1,50 +1,36 @@
...
-      @value={{readonly @options.goalValue}}
-      @focus-out={{fn this.updateGoal}}
-      @key-up={{fn this.updateGoal}}
+  </DenaliInputGroup>
+
+  <DenaliInputGroup @label="Goal" @isStacked= {{true}} class="goal-gauge-config__section-header">
+    <DenaliInput
+      @size="medium"
+      type="text"
+      class="goal"
```
Comment (jkusa) [inline packages/core/addon/components/navi-visualization-config/goal-gauge.hbs:30]:
```suggestion
      class="goal-gauge-config__goal-input"
```

File: packages/reports/addon/components/navi-column-config/base.hbs [hunk 7]
```diff
@@ -17,18 +17,21 @@
...
-      <Input
-        class="navi-column-config-base__column-name-input navi-column-config-base__option-input"
-        id="{{this.classId}}-column-name-input"
-        @value={{readonly @column.fragment.alias}}
-        @placeholder={{@column.fragment.displayName}}
-        @key-up={{fn @onRenameColumn}}
+  <div class="navi-column-config-base__option navi-column-config-base__column-name">
+    <DenaliInputGroup
+    @label="Column Name"
+    @isStacked= {{true}}
```
Comment (jkusa) [inline packages/reports/addon/components/navi-column-config/base.hbs:23]:
```suggestion
      @label="Column Name"
      @isStacked= {{true}}
```

File: packages/reports/addon/components/navi-column-config/base.hbs [hunk 8]
```diff
@@ -17,18 +17,21 @@
...
+    >
+      <DenaliInput
+        @size= "medium"
+        type="text"
+        class="navi-column-config-base__denali"
+        value={{readonly @column.fragment.alias}}
```
Comment (jkusa) [inline packages/reports/addon/components/navi-column-config/base.hbs:29]:
```suggestion
        value={{@column.fragment.alias}}
```

File: packages/reports/addon/components/navi-column-config/base.hbs [hunk 9]
```diff
@@ -17,18 +17,21 @@
...
-        @value={{readonly @column.fragment.alias}}
-        @placeholder={{@column.fragment.displayName}}
-        @key-up={{fn @onRenameColumn}}
+  <div class="navi-column-config-base__option navi-column-config-base__column-name">
+    <DenaliInputGroup
+    @label="Column Name"
+    @isStacked= {{true}}
+    >
+      <DenaliInput
+        @size= "medium"
```
Comment (jkusa) [inline packages/reports/addon/components/navi-column-config/base.hbs:26]:
```suggestion
        @size="medium"
```

File: packages/reports/addon/components/navi-column-config/base.hbs [hunk 10]
```diff
@@ -17,18 +17,21 @@
...
+        type="text"
+        class="navi-column-config-base__denali"
```
Comment (jkusa) [inline packages/reports/addon/components/navi-column-config/base.hbs:28]:
```suggestion
        class="navi-column-config-base__column-name-input navi-column-config-base__option-input"
```

File: packages/reports/addon/components/navi-column-config/base.hbs [hunk 11]
```diff
@@ -17,18 +17,21 @@
...
+        value={{readonly @column.fragment.alias}}
+        placeholder={{@column.fragment.displayName}}
+        {{on "input" (pipe (pick 'target.value') (fn @onRenameColumn ) )}}
```
Comment (jkusa) [inline packages/reports/addon/components/navi-column-config/base.hbs:31]:
```suggestion
        {{on "input" (pipe (pick 'target.value') (fn @onRenameColumn))}}
```

File: packages/reports/addon/components/navi-column-config/base.hbs [hunk 12]
```diff
@@ -17,18 +17,21 @@
...
       />
-    </div>
-
+    </DenaliInputGroup>
+ </div>
```
Comment (jkusa) [inline packages/reports/addon/components/navi-column-config/base.hbs:34]:
```suggestion
  </div>
```

File: packages/reports/app/styles/navi-reports/components/navi-column-config/base.less [hunk 13]
```diff
@@ -21,21 +21,10 @@
...
-      background-color: @denali-fields-default-bg;
-      border-radius: 2px;
-      flex: 1;
-      font-family: @font-family-sans-serif;
-      font-size: @font-size-mid-base;
-      min-height: 32px;
-      padding: 5px 10px;
-    }
+  &__denali {
+    width: 213px !important;
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/navi-column-config/base.less:27]: Does 100% work?

File: packages/reports/app/styles/navi-reports/components/navi-modal.less [hunk 14]
```diff
@@ -54,6 +54,10 @@
     line-height: inherit;
   }
 
+  &__denali-input {
+    width: 580px !important;
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/navi-modal.less:58]: Is this being used?

Comment (ayeswarya) [reply packages/reports/app/styles/navi-reports/components/navi-modal.less:58]: Not anymore. I added this for the modal inputs

File: packages/core/tests/acceptance/table-test.ts (hunk 2 above)
Comment (ayeswarya) [reply packages/core/tests/acceptance/table-test.ts:82]: I had to add this for the tests to work. I used the inspect element and table header cell input was nested within input

File: packages/core/tests/acceptance/table-test.ts (hunk 1 above)
Comment (ayeswarya) [reply packages/core/tests/acceptance/table-test.ts:81]: I had to add this for the tests to work. I used the inspect element and table header cell input was nested within input

File: packages/reports/addon/components/navi-column-config/base.hbs (hunk 10 above)
Comment (ayeswarya) [reply packages/reports/addon/components/navi-column-config/base.hbs:28]: Added wrapperclass

File: packages/core/addon/components/navi-visualization-config/metric-label.hbs [hunk 15]
```diff
@@ -1,18 +1,18 @@
...
+    @label="Label"
+    @isStacked={{true}}
+    class="metric-label-config__section-header"
+  >
+    <DenaliInput
+      @size="medium"
+      type="text"
+      class="metric-label-config__label-input"
+      placeholder="No Label"
+      value={{readonly this.metricColumn.displayName}}
```
Comment (jkusa) [inline packages/core/addon/components/navi-visualization-config/metric-label.hbs:13]:
```suggestion
      value={{this.metricColumn.displayName}}
```

File: packages/core/tests/integration/components/navi-visualization-config/metric-label-test.ts [hunk 16]
```diff
@@ -83,8 +83,8 @@ module('Integration | Component | visualization config/metric-label', function(h
 
     await render(Template);
 
-    await fillIn('.metric-label-config__description-input', 'foo');
-    await blur('.metric-label-config__description-input');
+    await fillIn('.metric-label-config__label-input', 'foo');
+    await blur('.metric-label-config__label-input');
```
Comment (jkusa) [inline packages/core/tests/integration/components/navi-visualization-config/metric-label-test.ts:87]:
```suggestion
```

**Review by jkusa — APPROVED** [review]

**Pull request merged**